/usr/lib/python2.7/dist-packages/Crypto/Cipher/blockalgo.py:141: FutureWarning: CTR mode needs counter parameter, not IV
  self._cipher = factory.new(key, *args, **kwargs)
--------------------------------------------------------------------------------------------
now start the host :10.11.5.75
[]
[u'/data/CI/kds-data_63_23210\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_63_23210/kds-data.sh start
--------------------------------------------------------------------------------------------
now start the host :10.11.5.75
[]
[u'/data/CI/kds-data_75_23260\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_75_23260/kds-data.sh start
--------------------------------------------------------------------------------------------
now start the host :10.11.5.75
[]
[u'/data/CI/kds-data_75_23270\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_75_23270/kds-data.sh start
--------------------------------------------------------------------------------------------
now start the host :10.11.5.73
[]
[u'/data/CI/kds-data_63_23210\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_63_23210/kds-data.sh start
--------------------------------------------------------------------------------------------
now start the host :10.11.5.73
[]
[u'/data/CI/kds-data_73_23260\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_73_23260/kds-data.sh start
--------------------------------------------------------------------------------------------
now start the host :10.11.5.73
[]
[u'/data/CI/kds-data_73_23270\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_73_23270/kds-data.sh start
--------------------------------------------------------------------------------------------
now start the host :10.11.5.74
[]
[u'/data/CI/kds-data_63_23210\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_63_23210/kds-data.sh start
--------------------------------------------------------------------------------------------
now start the host :10.11.5.74
[]
[u'/data/CI/kds-data_74_23260\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_74_23260/kds-data.sh start
--------------------------------------------------------------------------------------------
now start the host :10.11.5.74
[]
[u'/data/CI/kds-data_74_23270\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_74_23270/kds-data.sh start
--------------------------------------------------------------------------------------------
now start the host :10.11.5.76
[]
[u'/data/CI/kds-data_63_23210\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_63_23210/kds-data.sh start
--------------------------------------------------------------------------------------------
now start the host :10.11.5.76
[]
[u'/data/CI/kds-data_76_23260\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_76_23260/kds-data.sh start
--------------------------------------------------------------------------------------------
now start the host :10.11.5.76
[]
[u'/data/CI/kds-data_76_23270\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_76_23270/kds-data.sh start
--------------------------------------------------------------------------------------------
now start the host :10.11.5.78
[]
[u'/data/CI/kds-data_75_23260\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_75_23260/kds-data.sh start
--------------------------------------------------------------------------------------------
now start the host :10.11.5.78
[]
[u'/data/CI/kds-data_75_23270\n', u'kds-data.jar\n']
process is starting,wait a momunt
sudo /data/CI/kds-data_75_23270/kds-data.sh start
Traceback (most recent call last):
  File "/home/op/mesh_deploy_5.80/guizhou_start.py", line 32, in <module>
    ip , krs_sh, krs_jar, krs_jar_bak, lib_path ,server  = line.strip().split()
ValueError: need more than 0 values to unpack
[Finish] Failed to deploy [333]